src/components/ha-yaml-editor.ts

```diff
@@ -107,6 +107,7 @@ export class HaYamlEditor extends LitElement {
                 "ui.components.yaml-editor.copy_to_clipboard"
               )}
             </mwc-button>
+            <slot name="extra-actions"></slot>
```

Comment:
This one should be outside of the `copyClipboard` ternary

Reply:
Changed, is another property the best way to solve? I don't want to create the card-actions div if there aren't any actions..

Reply:
Yeah looks good! 👍